core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1144,13 +1218,19 @@ class SparkContext(config: SparkConf) extends Logging {
   }
 
   /**
-   * Get an RDD for a Hadoop SequenceFile with given key and value types.
+   * Get an RDD for a Hadoop `SequenceFile` with given key and value types.
    *
-   * @note Because Hadoop's RecordReader class re-uses the same Writable object for each
-   * record, directly caching the returned RDD or directly passing it to an aggregation or shuffle
-   * operation will create many references to the same object.
-   * If you plan to directly cache, sort, or aggregate Hadoop writable objects, you should first
-   * copy them using a `map` function.
+   * @note because Hadoop's `RecordReader` class re-uses the same `Writable` object for each
```

Comment:
[Spark style guide](http://spark.apache.org/contributing.html) doesn't contain anything about continuation indentation and refers to the [Scala's own style guide](http://docs.scala-lang.org/style/scaladoc.html) which shows that indentation should be used.